linux/Jenkinsfile

```diff
@@ -21,13 +33,15 @@ pipeline {
         timeout(time: 2, unit: 'HOURS')
     }
     parameters {
+        choice(name: 'TYPE', choices: Types.ALL, description: 'Build either JDK, or CA Certificates\n  CA Certificates only on Debian possible (ARCH/VERSION irrelevant)')
```

Comment:
I was happy to be able to test everything on Jenkins as well. There is some more overlap between Jenkins and GH actions though.

Reply:
Where possible we should continue to publish using GitHub actions as it avoids running build jobs on the Jenkins infrastructure we pay for. I'll let other people share their thoughts but we should either accept this and remove the GitHub action publish job or remove this. As this has nothing to do with JRE's it might be worth cherry-picking this out into it's own pull request?

Reply:
Well, the type parameter was necessary to either build Jdk or Jre on Jenkins. I agree the build of the CA-Certificates was not necessary here.

But then, the build of JDK and JRE on Jenkins is not necessary either as both can be build (and are built) on GH as well?

Reply:
the reason we build the JDK/JRE's on Jenkins is that the Debian build needs to be run on different architectures (e.g can't be cross-compiled). If you look at the way the GH action runs, it only tests x64 for Debian but does all architectures for RH/Suse. My preference going forward would be to handle the JDK/JRE builds completely in GitHub actions if possible (we'd just need to solve the issue with cross-compiling the Debian files (this could be a great feature if you're willing to implement it). Either way as this change is in no way related to the JRE implementation, I'd prefer it if you removed the Jenkinsfile patches that added support for building the CACerts.